fetch.bs

```diff
@@ -4910,49 +4912,33 @@ constructor must run these steps:
    <a for=request>use-CORS-preflight flag</a>.
   </ol>
 
- <li><p>Set <var>r</var>'s <a for=Request>request</a>'s
- <a for=request>body</a> to <var>inputBody</var>.
  <!-- Any steps after this must not throw. -->
 
- <li><p>Set <var>r</var>'s <a for=Body>MIME type</a> to
- the result of <a lt="extract a MIME type" for="header list">extracting a MIME type</a>
- from <var>r</var>'s <a for=Request>request</a>'s
- <a for=request>header list</a>.
-
  <li>
-  <p>If <var>input</var> is a {{Request}} object and
-  <var>input</var>'s <a for=Request>request</a>'s
-  <a for=request>body</a> is non-null, run these substeps:
+  <p>If <var>inputBody</var> is non-null, then run these substeps:
 
   <ol>
-   <li><p>Let <var>dummyStream</var> be an <a>empty</a>
-   {{ReadableStream}} object.
-
-   <li><p>Set <var>input</var>'s <a for=Request>request</a>'s
-   <a for=request>body</a> to a new <a for=/>body</a> whose
-   <a for=body>stream</a> is <var>dummyStream</var>.
-
    <li>
-    <p>Let <var>reader</var> be the result of <a lt="get a reader" for=ReadableStream>getting a reader</a>
-    from <var>dummyStream</var>.
-    <p class="note no-backref">This operation will not throw an exception.
-
-   <li>
-    <p><a lt="read all bytes" for=ReadableStream>Read all bytes</a> from
-    <var>dummyStream</var> with <var>reader</var>.
-    <p class="note no-backref">This operation makes <var>dummyStream</var> disturbed.
+    <p>Let <var>rs</var> bs a {{ReadableStream}} object from which one can read the exactly
+    same data as one could read from <var>inputBody</var>'s <a for=body>stream</a>.
+    <p class="note no-backref">This will be specified more precisely once
+    <a href=https://streams.spec.whatwg.org/#ts-model>transform stream</a> and
+    <a href=https://streams.spec.whatwg.org/#pipe-chains>piping</a> are precisely defined.
+    This makes <var>inputBody</var>'s <a for=body>stream</a> <a for=ReadableStream>locked</a> and
+    <a for=ReadableStream>disturbed</a> immediately.
```

Comment:
Thanks, this PR looks good to me now, but we will need tests per the new process.